src/lit-html.ts

```diff
@@ -343,15 +339,17 @@ export class NodePart extends Part {
     }
   }
 
-  clear() {
+  clear(startNode :Node = this.startNode) {
     this._previousValue = undefined;
-    const range = document.createRange();
-    range.setStartAfter(this.startNode);
-    range.setEndBefore(this.endNode);
-    range.deleteContents();
-    range.detach();
-  }
 
+    let node = startNode.nextSibling!;
```

Comment:
I think rather than a default parameter, you can do this to get the old behavior:

```ts
clear(startNode?: Node) {
  let node = startNode === undefined ? this.startNode.nextSibling! : startNode;
  ...
}
```